src/test/java/net/sf/jabref/model/entry/BibEntryTests.java

```diff
@@ -212,16 +212,21 @@ public void setNullField() {
 
     @Test(expected = NullPointerException.class)
     public void addNullKeywordThrowsNPE() {
-        keywordEntry.addKeyword(null);
+        keywordEntry.addKeyword(null, Globals.prefs.get(JabRefPreferences.KEYWORD_SEPARATOR));
```

Comment:
That basically means that there shouldn't be any logic tests which initializes `Globals.prefs`. The question is just how long we should enforce that... I can remove the ones here, though.